lib/devices/shelly-uni.js

```diff
@@ -0,0 +1,32 @@
+const { Switch } = require('./base')
+
+class ShellyUni extends Switch {
+  constructor(id, host) {
+    super(id, host)
+
+    this._defineProperty('relay0', [112, 1101], false, Boolean)
+    this._defineProperty('relay1', [122, 1201], false, Boolean)
+
+    this._defineProperty('input0', [118, 2101], 0, Number)
+    this._defineProperty('inputEvent0', [2102], '', String)
+    this._defineProperty('inputEventCounter0', [2103], 0, Number)
+    this._defineProperty('input1', [128, 2201], 0, Number)
+    this._defineProperty('inputEvent1', [2202], '', String)
+    this._defineProperty('inputEventCounter1', [2203], 0, Number)
+
+    this._defineProperty('externalTemperature0', [119, 3101], null, Number)
+    this._defineProperty('externalTemperature1', [3201], null, Number)
+    this._defineProperty('externalTemperature2', [3301], null, Number)
+    this._defineProperty('externalTemperature4', [3401], null, Number)
```

Comment:
What happened to `externalTemperature3`?

Reply:
I messed up. Bare with me, not a daily Github user here.